docs/Manpage.md

```diff
@@ -1056,6 +1056,19 @@ directory.
 * `--commit`:
   Commit changes to the SPDX license data.
 
+### `update-python-resources` [*`options`*] *`formula`*
+
+Update versions for PyPI resource blocks in *`formula`*.
+
+* `-p`, `--print-only`:
+  Print the updated resource blocks instead of changing *`formula`*.
+* `-s`, `--silent`:
+  Suppress any output.
+* `--ignore-non-pypi-packages`:
+  Don't fail if *`formula`* is not a PyPI package.
+* `--constraints`:
+  Append these constraints when performing dependency resolution (e.g. `--constraints='cffi>1.11.3'`)
```

Comment:
Not sure if worth noting, but passing cffi here would result in latest cffi being required, whether the formula in question wants it or not, so in pypa terms this is a requirement instead of a constraint. Potentially confusing with pip's `--constraint`, which wouldn't install the constraint if it wasn't required.

Reply:
Thanks for taking a look at this, @ddelange

@jonchang I'll defer to you for the use case here. Originally this was suggested as a temporary workaround for https://github.com/ddelange/pipgrip/issues/25 (which has since been incorporated into a release), but maybe there isn't a good enough use case to have it as an available flag.

(changing the name of the flag is not an issue from my point of view)

Reply:
Yeah, I think the escape hatch is no longer really needed since we've figured out the `cffi` issue and will probably figure out the `azure-cli` issue above. If you haven't identified other formulae that need the additional constraint we could probably just drop it.

Reply:
I haven't at this point, so I'll remove it. If we find any more in the future, we can always re-add it or deal with it upstream if it's a `pipgrip` issue.

Thanks again, @ddelange for being so helpful and responsive with this!